FoxHound.Client/src/Navigation/NavDrawer.tsx

```diff
@@ -0,0 +1,211 @@
+import React from "react";
+import clsx from "clsx";
+import {
+  makeStyles,
+  useTheme,
+  Theme,
+  createStyles,
+} from "@material-ui/core/styles";
+import Drawer from "@material-ui/core/Drawer";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import List from "@material-ui/core/List";
+import Typography from "@material-ui/core/Typography";
+import Divider from "@material-ui/core/Divider";
+import IconButton from "@material-ui/core/IconButton";
+import MenuIcon from "@material-ui/icons/Menu";
+import ChevronLeftIcon from "@material-ui/icons/ChevronLeft";
+import ChevronRightIcon from "@material-ui/icons/ChevronRight";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import ListItemText from "@material-ui/core/ListItemText";
+import VisibilityIcon from "@material-ui/icons/Visibility";
+import EditIcon from "@material-ui/icons/Edit";
+import DesktopWindowsIcon from "@material-ui/icons/DesktopWindows";
+import { Link } from "react-router-dom";
+import BathtubIcon from "@material-ui/icons/Bathtub";
+import EmojiPeopleIcon from "@material-ui/icons/EmojiPeople";
+import PostAddIcon from "@material-ui/icons/PostAdd";
```

Comment:
Most of these can be combined into a single import. All of the @material-ui/core can  be combined and I think all of the @material-icons can be combined (although, i'm not as sure on the icons)

Reply:
I was able to combine all of the @material-ui/core imports; however, I was not able to figure out a way of combining the Icons.  Wasn't sure if there was another approach that I could take in order to clean up the icon imports.  According to the documentation on the Material site, it looks like it should be possible with some babel configuration...

Reply:
Probably not worth it then for the icons...not a problem leaving them as separate imports